### ISSUE TITLE
Document dependencies for a for minimal X11 build

### ISSUE THREAD

**Issue by croth1** [conda-forge/conda-forge.github.io#776]
We have not documented yet the dependencies for a working X11 build. There should be xorg packages for all linux dependencies, so there should be no need for CDT packages.

xref: https://github.com/conda-forge/matplotlib-feedstock/pull/202

**Comment by ocefpaf** [comment]
I don't think we ever made a decision on this actually. The CDT are used for linking only and X11 are either for testing (when we are sticking to what defaults does) or for building too. However, I know that some people do not recommend using conda-forge's X11 for building too. I'll bring this up again in our next core meeting. Let me know if you want to be there for this discussion.
